app/controllers/application_controller.rb

```diff
@@ -49,8 +49,16 @@ def append_info_to_payload(payload)
   attr_writer :analytics
 
   def analytics
-    @analytics ||=
-      Analytics.new(user: analytics_user, request: request, sp: current_sp&.issuer, ahoy: ahoy)
+    return @analytics if @analytics
+    update_session_paths_visited_for_analytics
+    @analytics =
+      Analytics.new(
+        user: analytics_user,
+        request: request,
+        sp: current_sp&.issuer,
+        is_new_session_path: first_path_visit_this_session?,
```

Comment:
I meant changing this key too, for consistency, and I think it would be clearer for future people looking at these logs

```suggestion
        first_path_visit_this_session: first_path_visit_this_session?,
```